README.md

```diff
@@ -158,9 +160,9 @@ gem 'simplecov' # test
 
 ```shell
 gem install bundler
-# prerequisites postgresql-XX-client library on OS (XX=major release installed in OS)
+# prerequisites libraries on Ubuntu OS:
+# sudo apt install postgresql-client-13 libpq-dev libsystemd-dev
 bundle config set without 'libvirt ovirt mysql2'
```

Comment:
mysql2 is long gone: it was the reason for the major version bump in Foreman 2.x:
```suggestion
bundle config set without 'libvirt ovirt'
```